tpm2/tpm2_test.go

```diff
@@ -1023,6 +1009,11 @@ func TestNVReadWrite(t *testing.T) {
 		attr                = AttrOwnerWrite | AttrOwnerRead
 	)
 
+	// Undefine the space, just in case previous test failed to.
```

Comment:
I'm not sure I understand the comment. After removing the TestNVRead case, the previous test is TestCreateAndCertifyCreation, and I don't see an explicit NVUndefineSpace there. Please clarify.

Reply:
I meant "the previous run of this same test". Clarified in the comment.